### PR TITLE
Increase Slat Angle Sampling Frequency for Blind Properties

### PR DESCRIPTION
**Pull request by amirroth**
[This issue](https://github.com/NREL/EnergyPlus/issues/10640) summarizes the underlying problem and its solution. TL;DR fixed-slat and movable-slat blinds used two different calculation methods for blind properties, direct calculation at the slat angle for fixed and interpolation between pre-calculated values sampled at 10 degree intervals for movable. This was found to cause meaningful differences between fixed-slat blinds and movable-slat blinds with the slats in the same position (specifically and especially the default 45 degrees). Some notable differences remained when the sampling period was reduced to 5 degrees, and so the period was further reduced to 1 degree. 

"Errors" are expected in the three test files that use movable-slat blinds. 

This issue was uncovered during the course of a refactor. The resolution is being packaged in a separate PR in order to isolate the output changes.

Comments and follow-up from @mjwitte and @vidanovic in the issue itself.

### CODE REVIEW

File: src/EnergyPlus/Material.hh [hunk 1]
```diff
@@ -61,7 +61,7 @@ namespace EnergyPlus {
 
 namespace Material {
 
-    constexpr int MaxSlatAngs(19);
+    constexpr int MaxSlatAngs(181); // 1 degree increments for slat angles (We'll see what the performance implications are)
```
Comment (amirroth) [inline src/EnergyPlus/Material.hh:64]: This is the only real change.

File: src/EnergyPlus/WindowManager.cc [hunk 2]
```diff
@@ -5833,7 +5833,7 @@ namespace Window {
         DPhi = 5.0 * Constant::DegToRadians;
 
         // Integrate from -90 to 0 deg
-        for (int IPhi = 1; IPhi <= 18; ++IPhi) {
+        for (int IPhi = 1; IPhi <= (Material::MaxProfAngs / 2); ++IPhi) {
```
Comment (amirroth) [inline src/EnergyPlus/WindowManager.cc:5836]: Noticed that these are literals so changed to use the `constexpr int MaxProfAngs`. That is all.

**Comment by Myoldmopar** [comment]
I'll run clang format here and get this merged in.

**Comment by Myoldmopar** [comment]
Looks happy enough, let's merge this.  Thanks @amirroth

**Pull request merged**